src/themis/secure_keygen.c

```diff
@@ -74,6 +84,7 @@ themis_status_t themis_gen_key_pair(soter_sign_alg_t alg,
     themis_status_t private_result = THEMIS_FAIL;
     themis_status_t public_result = THEMIS_FAIL;
     soter_sign_ctx_t* ctx = NULL;
+    bool compressed = true;
```

Comment:
can we leave comment that it is default value due to historical reasons? to understand that it is not most efficient mode format, just because backward compatibility support?

Reply:
> can we leave comment that it is default value due to historical reasons?

Here you go.